src/config.js

```diff
@@ -33,8 +33,10 @@ export const AUTH0_API = {
 // const apiUrl = 'https://www.rpc.rambadvetclinic.com';
 
 // API Development
-const apiUrl = 'https://www.uatradhiyan.rambadvetclinic.com';
+// const apiUrl = 'https://www.uatradhiyan.rambadvetclinic.com';
 
+// API LOCAL
+const apiUrl = 'http://localhost:8000';
```

Comment:
eh ini jgn lupa di ganti ya, klo di local gw kan nembaknya tetep kesini 'https://www.uatradhiyan.rambadvetclinic.com';. engga run BE di local

Reply:
wait deh, aku setup ,env nya

Reply:
okay aku udah tambahin .env mas, kalo 
REACT_APP_MODE=local bakalan ke localhost:8000 tapi kalo ga di set default ke server development